crates/shader-crate-template/Cargo.toml

```diff
@@ -15,8 +15,77 @@ spirv-std = { git = "https://github.com/Rust-GPU/rust-gpu", rev = "82a0f69" }
 [target.'cfg(target_arch = "spirv")'.dependencies]
 glam = { version = "0.29", default-features = false, features = ["libm"] }
 
-
 # Dependencies for CPU code
 [target.'cfg(not(target_arch = "spirv"))'.dependencies]
 glam = { version = "0.29", features = ["std"] }
 
+[package.metadata.rust-gpu.build]
+# Where to output the compiled shader. Defaults to where `cargo gpu` is called from.
+# TODO: Should it default to the root of the shader crate?
```

Comment:
It probably should default to the root of the shader crate, but we can keep that for later.